src/guest_webapp/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+# not sure which version to use
+version: '3.8'
+
+services:
+  react-frontend:
+    build:
+      context: .
+      # dockerfile: Dockerfile
+    ports:
+      - "5173:5173"
+    volumes:
```

Comment:
NOT ACTIONABLE:
I could be wrong, but this is only used for development. When we deploy this, it will need to be changed.